attributes/default.rb

```diff
@@ -4,6 +4,7 @@
 default.loggly.tags = []
 
 default.loggly.log_files = []
+default.loggly.apps = {}
```

Comment:
Is apps synonymous with rsyslog versions? I wonder if we could think of a more descriptive name.